app/javascript/controllers/tab_nav_controller.js

```diff
@@ -0,0 +1,24 @@
+import { Controller } from '@hotwired/stimulus'
+import * as bootstrap from 'bootstrap'
+
+export default class extends Controller {
+  static values = { selector: String }
+
+  showNext (event) {
+    event.preventDefault()
+
+    const nextTab = this.currentTab.nextElementSibling
+    bootstrap.Tab.getOrCreateInstance(nextTab).show() // eslint-disable-line no-undef
```

Comment:
Did you find you needed the eslint disables after having the `import` on line 2?

Reply:
No, not needed. Removing.